llvm/tools/llvm-cas-dump/MCCASPrinter.h

```diff
@@ -0,0 +1,47 @@
+//===- llvm-cas-object-format.cpp - Tool for the CAS object format --------===//
```

Comment:
Comment is out of sync

Reply:
My bad, I thought I had updated all of these